### ISSUE TITLE
network parameters used in MVX-Net model

### ISSUE THREAD

**Issue by youssefAlaaThabet** [open-mmlab/mmdetection3d#1484]
What are the network parameters used for the training of the MVX-Net model ?

**Comment by Tai-Wang** [comment]
Do you mean the config for training MVX-Net model? Please refer to [this folder](https://github.com/open-mmlab/mmdetection3d/tree/master/configs/mvxnet) for configs of MVX-Net.

**Comment by youssefAlaaThabet** [comment]
yes, what was the optimizer used ? the learning rate ? number of epochs ? , batch size ? the total training time ?

**Comment by youssefAlaaThabet** [comment]
I really need  this information for my thesis. 
Hopefully you could help me figure them out .

**Comment by Tai-Wang** [comment]
Sorry for the late reply. You can see it inherits from two configs namely "cosine" and "default runtime". Please find the information you want in those configs.
